src/qos_enclave/Cargo.lock

```diff
@@ -1664,6 +2462,12 @@ dependencies = [
  "windows-sys 0.52.0",
 ]
 
+[[package]]
+name = "spin"
+version = "0.9.8"
```

Comment:
Have not reviewed this fully, but looks alright: https://github.com/zesterer/spin-rs